### PR TITLE
feat(Scripts/Commands): .aura stack - Modify aura stacks

### PR DESCRIPTION
**Pull request by avarishd**
Yet another custom change...

## Changes Proposed:

This PR proposes changes to:
-  [x] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [ ] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

.aura 44833
.aura stack 44833 10
.list auras
.aura stack 44833 -5
.list auras

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Kitzunu** [comment]
`.aura stack $spellId` would be preffered imo

File: data/sql/updates/pending_db_world/rev_1721586059640959700.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+--
+DELETE FROM `command` WHERE `name`='aurastack';
+INSERT INTO `command` VALUES ('aurastack', 2, 'Syntax: .aurastack #spellid #stacks\r\n\r\nModify #stacks of an already applied #spellid to the selected Unit.');
+
+DELETE FROM `acore_string` WHERE `entry` IN (373,374);
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES
+(373, 'Target unit does not have aura %u!'),
+(374, 'No stack amount specified!');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1721586059640959700.sql:8]: This last one I dont see a need for really, invalid syntax would suffice

File: src/server/scripts/Commands/cs_misc.cpp [hunk 2]
```diff
@@ -653,6 +659,51 @@ class misc_commandscript : public CommandScript
...
+        }
+
+        Aura* aur = target->GetAura(spell->Id);
+        if (!aur)
+        {
+            handler->SendErrorMessage(LANG_COMMAND_AURASTACK_NO_AURA, spell->Id);
+            return false;
+        }
+
+        if (aur && !spell->StackAmount)
```
Comment (Kitzunu) [inline src/server/scripts/Commands/cs_misc.cpp:696]: aur check here is redundant

File: data/sql/updates/pending_db_world/rev_1721586059640959700.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+--
+DELETE FROM `command` WHERE `name`='aura stack';
+INSERT INTO `command` VALUES ('aura stack', 2, 'Syntax: .aurastack #spellid #stacks\r\n\r\nModify #stacks of an already applied #spellid to the selected Unit.');
+
+DELETE FROM `acore_string` WHERE `entry` IN (373,374,375);
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES
+(373, 'Target unit does not have aura %u!'),
+(374, 'No stack amount specified!'),
+(375, 'Spell %u cannot have stacks!');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1721586059640959700.sql:9]:
```suggestion
INSERT INTO `acore_string` (`entry`, `content_default`) VALUES
(373, 'Target unit does not have aura {}!'),
(374, 'No stack amount specified!'),
(375, 'Spell {} cannot have stacks!');
```

Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1721586059640959700.sql:9]: not a single string in acore_string is using curly brackets.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1721586059640959700.sql:9]: wrong

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1721586059640959700.sql:9]: https://github.com/azerothcore/azerothcore-wotlk/commit/3fbded8f6bdaae1ec6897dae34afdab36b124899

Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1721586059640959700.sql:9]: sneaky

**Review by IntelligentQuantum — APPROVED** [review]

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**